systemd/ubuntu-advantage.service

```diff
@@ -0,0 +1,26 @@
+# This service only runs on GCP to enable auto-attaching to Ubuntu Advantage
+# services when an Ubuntu Pro license is added to a GCP machine.
+# If you are uninterested in the (free for personal use) Ubuntu Advantage
+# services, including security updates after standard EOL and kernel patching
+# without rebooting, then you can safely stop and disable this service:
+# sudo systemctl stop ubuntu-advantage.service
+# sudo systemctl disable ubuntu-advantage.service
+
+[Unit]
+Description=Ubuntu Advantage GCP Auto Attach Daemon
+Documentation=man:ubuntu-advantage https://ubuntu.com/advantage
+After=network.target network-online.target systemd-networkd.service ua-auto-attach.service cloud-config.target
+
+# Only run if not already attached
+ConditionPathExists=!/var/lib/ubuntu-advantage/private/machine-token.json
+# Only run on GCP
+ConditionPathExists=/run/cloud-init/cloud-id-gce
```

Comment:
SRU review: Am I correct in understanding that as per the [spec document](https://docs.google.com/document/d/1xeOnO1Uh1f4pHx3DlQOGTor0b60zH5gzvx4MZwhwltI) this is pending a new cloud-init feature?

Not blocking, just checking.